candies/cli/parsers/parser.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABCMeta, abstractmethod
+from typing import List, Dict, Any
+
+
+class Parser(metaclass=ABCMeta):
+    """A parser for command line arguments."""
+
+    @abstractmethod
+    def parse(self, args: List[str]) -> Dict[str, Any]:
```

Comment:
So, if the return type of the `parse` function is `Dict[str, Any]`, does this mean that [the problem of parser's non-determinism](https://github.com/candy-kingdom/cli/issues/9#issuecomment-579723033) will be handled by passing the function to it?

Reply:
We've discussed this thing and decided to enhance the parser API later. It'll return the list of nodes.